.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+name: IntuneAppBuilder Deployment
+
+on:
+  schedule:
+    - cron: "30 7 * * *"
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - master
+
+env:
+  BuildConfiguration: 'Release'
+  Verbosity: 'normal'
+  ArtifactName: 'Artifacts'
+  ArtifactDirectory: './artifacts/'
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  Build:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.301
+
+      - name: dotnet restore
+        run: dotnet restore --verbosity $env:Verbosity
+      
+      - name: dotnet build
+        run: dotnet build --configuration $env:BuildConfiguration --verbosity $env:Verbosity --no-restore
+
+      - name: dotnet pack
+        run: dotnet pack --configuration $env:BuildConfiguration --verbosity $env:Verbosity --no-build -o $env:ArtifactDirectory
+
+      - name: dotnet test
+        run: dotnet test --configuration $env:BuildConfiguration --no-build --verbosity $env:Verbosity
+        env:
+          AadAuth:Password: ${{ secrets.TEST_AADAUTH_PASSWORD }}
+          AadAuth:Username: ${{ secrets.TEST_AADAUTH_USERNAME }}
+
+      - name: Publish pipeline artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.ArtifactName }}
+          path: ${{ env.ArtifactDirectory }}
+
+  DeployNuGet:
+    needs: Build
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
you don't need to do a checkout in the Deploy step do you?

Reply:
Strange... I thought you already commented this and I removed it.